package.json

```diff
@@ -61,7 +61,9 @@
 		"prern-bundle": "patch-package --patch-dir gutenberg/packages/react-native-editor/metro-patch",
 		"rn-bundle": "react-native bundle",
 		"postrn-bundle": "patch-package --reverse --patch-dir gutenberg/packages/react-native-editor/metro-patch",
-		"bundle": "npm run clean; npm run core prebundle && npm run bundle:js && npm run genstrings",
+		"i18n:gutenberg": "./bin/i18n-download.sh gutenberg",
```

Comment:
In the future, once the missing strings issue is solved, we would change this command to `cross-env REFRESH_I18N=1 npm run i18n:gutenberg`. This way we would enforce the cache update when generating the bundle.